R/Prior.R

```diff
@@ -324,17 +338,17 @@ prior_beta <- function(a, b, init = a / (a + b)) {
 
 #' Only Initial Values Specification
 #'
-#' @inheritParams Prior-Shared
 #' @family Prior
 #'
 #' @export
-prior_none <- function(init = 0.00001) {
+prior_none <- function() {
     Prior(
         parameters = list(),
         display = "<None>",
         repr_model = "",
         repr_data = "",
-        init = init,
+        sample = \(n) local_runif(n, -4, 4),
```

Comment:
also here wondering if those magic numbers would better live in a package option

Reply:
ah ok, I just think it would be more elegant to capture this in a package option, that defaults to `c(-4, 4)` but in principle the user could change it